catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/transformer/GmdTransformer.java

```diff
@@ -0,0 +1,302 @@
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/descriptiveKeywords/MD_Keywords/keyword/CharacterString");
+
+    private static final Path TOPIC_CATEGORY_PATH = new Path(
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/topicCategory/MD_TopicCategoryCode");
+
+    private static final Path BBOX_WEST_LON_PATH = new Path(
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/extent/EX_Extent/geographicElement/EX_GeographicBoundingBox/westBoundLongitude/Decimal");
+
+    private static final Path BBOX_EAST_LON_PATH = new Path(
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/extent/EX_Extent/geographicElement/EX_GeographicBoundingBox/eastBoundLongitude/Decimal");
+
+    private static final Path BBOX_SOUTH_LAT_PATH = new Path(
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/extent/EX_Extent/geographicElement/EX_GeographicBoundingBox/southBoundLatitude/Decimal");
+
+    private static final Path BBOX_NORTH_LAT_PATH = new Path(
+            "/MD_Metadata/identificationInfo/MD_DataIdentification/extent/EX_Extent/geographicElement/EX_GeographicBoundingBox/northBoundLatitude/Decimal");
+
+    private static final List<Path> PATHS = Arrays.asList(FILE_IDENTIFIER_PATH,
+            DATE_STAMP_PATH,
+            CODE_LIST_VALUE_PATH,
+            CRS_AUTHORITY_PATH,
+            CRS_VERSION_PATH,
+            CRS_CODE_PATH,
+            TITLE_PATH,
+            ABSTRACT_PATH,
+            FORMAT_PATH,
+            LINKAGE_URI_PATH,
+            KEYWORD_PATH,
+            TOPIC_CATEGORY_PATH,
+            BBOX_WEST_LON_PATH,
+            BBOX_EAST_LON_PATH,
+            BBOX_SOUTH_LAT_PATH,
+            BBOX_NORTH_LAT_PATH);
+
+    private static final String TRANSFORM_EXCEPTION_MSG =
+            "Unable to transform from GMD Metadata to Metacard";
+
+    private final XStream xstream;
+
+    public GmdTransformer() {
+        QNameMap qmap = new QNameMap();
+        qmap.setDefaultNamespace(GmdMetacardType.GMD_NAMESPACE);
+        qmap.setDefaultPrefix("");
+        StaxDriver staxDriver = new StaxDriver(qmap);
+
+        xstream = new XStream(staxDriver);
+        xstream.setClassLoader(this.getClass()
+                .getClassLoader());
+        XstreamPathConverter converter = new XstreamPathConverter();
+        converter.setPaths(PATHS);
+        xstream.registerConverter(converter);
+        xstream.alias("MD_Metadata", XstreamPathValueTracker.class);
+
+    }
+
+    @Override
+    public Metacard transform(InputStream inputStream)
+            throws IOException, CatalogTransformerException {
+        return transform(inputStream, null);
+    }
+
+    @Override
+    public Metacard transform(InputStream inputStream, String id)
+            throws IOException, CatalogTransformerException {
+
+        XstreamPathValueTracker pathValueTracker = null;
+        String xml = IOUtils.toString(inputStream);
+        IOUtils.closeQuietly(inputStream);
```

Comment:
Should closeQuietly() be put in a finally block to make sure it always gets closed?